tests/e2e/steps_permissionless_ics.go

```diff
@@ -142,9 +142,10 @@ func stepsPermissionlessICS() []Step {
 						SpawnTime:     0,
 					},
 					PowerShapingParams: &PowerShapingParameters{
-						TopN:      0,
-						Allowlist: []string{getDefaultValidators()[ValidatorID("carol")].ValconsAddress},
-						Denylist:  []string{getDefaultValidators()[ValidatorID("bob")].ValconsAddress},
+						TopN:         0,
+						Allowlist:    []string{getDefaultValidators()[ValidatorID("carol")].ValconsAddress},
+						Denylist:     []string{getDefaultValidators()[ValidatorID("bob")].ValconsAddress},
+						Prioritylist: []string{getDefaultValidators()[ValidatorID("carol")].ValconsAddress},
```

Comment:
This change does not really do anything, right? Carol is already the only validator on the allowlist
If so, let's not have this change in; it just makes less clear what is actually tested imo

Reply:
I agree. I’ve removed it now.